cypress/test/buttons/ICSFileButton.cy.js

```diff
@@ -0,0 +1,26 @@
+import { mount } from 'cypress/react';
+import GoogleCalendarButton from '../../../components/buttons/ICSFileButton';
+
+describe('GoogleCalendarButton', () => {
+
+  it('renders correctly with custom props', () => {
+    const customHref = 'https://example.com';
+  
+    mount(
+      <GoogleCalendarButton
+        text="Custom Text"
+        href={customHref}
+        target="_self"
+        iconPosition="right"
+      />
+    );
+    cy.contains('Custom Text').should('be.visible');
+    cy.get('[ data-testid="Button-link"]').should('have.attr', 'href', customHref);
+    cy.get('[ data-testid="Button-link"]').should('have.attr', 'target', '_self');
```

Comment:
why are you adding space here? between `[  data-test` @reachaadrika 
Please install a formatter extension in your vscode


fix this space issue with all over places

Reply:
fixed